app/presenters/filtered_editions_presenter.rb

```diff
@@ -3,18 +3,64 @@
 class FilteredEditionsPresenter
   ITEMS_PER_PAGE = 20
 
-  def initialize(states_filter: [], assigned_to_filter: nil, format_filter: nil, title_filter: nil, page: nil)
+  def initialize(states_filter: [], assigned_to_filter: nil, content_type_filter: nil, title_filter: nil, page: nil)
     @states_filter = states_filter || []
     @assigned_to_filter = assigned_to_filter
-    @format_filter = format_filter
+    @content_type_filter = content_type_filter
     @title_filter = title_filter
     @page = page
   end
 
+  def title
+    @title_filter
+  end
+
+  def content_types
+    types = []
+
+    content_type_filter_selection_options.map do |format|
```

Comment:
We should probably rename `format` to `content_type` for consistency with the other changes.